src/vm-memory/src/bitmap.rs

```diff
@@ -0,0 +1,144 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+//! Atomic bitmap implementation.
+
+// Temporarly disable unused warnings.
+// TODO: remove these once the Bitmap integration is completed.
+#![allow(dead_code)]
```

Comment:
Isn't the Bitmap integration complete? Please remove this as it can hide issues going further